app/models/manageiq/providers/cloud_manager/metrics_capture.rb

```diff
@@ -0,0 +1,5 @@
+class ManageIQ::Providers::CloudManager::MetricsCapture < ManageIQ::Providers::BaseManager::MetricsCapture
+  def capture_ems_targets(options = {})
+    Metric::Targets.capture_cloud_targets([ems], options)
```

Comment:
@kbrock is the goal to move all of e.g. `capture_cloud_targets` into the `CloudManager::MetricsCapture` class?